config/config.yml

```diff
@@ -34,3 +34,7 @@ p2p:
     # jorropo
     - maddr: "/ip4/51.75.127.200/tcp/4141/p2p/12D3KooWPwRwwKatdy5yzRVCYPHib3fntYgbFB4nqrJPHWAqXD7z"
     - maddr: "/ip4/51.75.127.200/udp/4141/quic/p2p/12D3KooWPwRwwKatdy5yzRVCYPHib3fntYgbFB4nqrJPHWAqXD7z"
+  relayHack:
```

Comment:
what about just naming it `relays`, since they can be used in other valid circumstances too?